packages/frontend/src/components/EpochKeyEvent.jsx

```diff
@@ -1,41 +1,21 @@
-import React, { useContext, useEffect } from 'react'
+import React from 'react'
 import { observer } from 'mobx-react-lite'
-import state from '../contexts/state'
 import dayjs from 'dayjs'
 import relativeTime from 'dayjs/plugin/relativeTime'
 import './eventCard.css'
 
 dayjs.extend(relativeTime)
 
 export default observer(({ attestation }) => {
-  const { attesterId, epoch, posRep, negRep, graffiti, timestamp } = attestation
-  const { attester } = useContext(state)
-  useEffect(() => {
-    const loadData = async () => {
-      await attester.loadEpochsByAttester(attesterId)
-    }
-    loadData()
-  }, [])
-  const attesterEpoch = attester.epochsByNumber.get(epoch)
+  const { hash, posRep, negRep, graffiti, timestamp } = attestation
 
   return (
     <div className="event-card">
-      {attesterEpoch ? (
-        attesterEpoch.sealed ? (
-          <p>sealed</p>
-        ) : (
-          <p>not sealed</p>
-        )
-      ) : (
-        'Loading...'
-      )}
-      <p style={{ minWidth: '80px' }}>
-        {posRep - negRep}
-        <span style={{ fontSize: '12px', fontWeight: '600' }}>
-          <span style={{ color: 'green' }}>+{posRep}</span>/
-          <span style={{ color: 'red' }}>-{negRep}</span>
-        </span>
+      <p>
+        {hash.slice(0, 5)}...{hash.slice(-5)}
       </p>
+      <p style={{ color: 'green', fontWeight: '600' }}>+{posRep}</p>
+      <p style={{ color: 'red', fontWeight: '600' }}>-{negRep}</p>
```

Comment:
here separating pos and neg rep into 2 columns; another column here for total rep of attestation seems unnecessary.

@vivianjeng asked for the transaction hash and link to etherscan here..
I've added the column and the `hash` from the attestation object. I see that this is not a tx hash, but unsure how to access those. Will this need to be added to the db similar to the way we added the timestamps?

Reply:
Sorta, but we don't need to make an async request. We get the transaction hash on the event so we can synchronously add it. I think we can do a collection keying document `_id`s to transaction hashes. I'm not sure if we should include transaction hashes at the core level or not. It's hard to tell what we need just for the explorer or what is generally useful.